cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -50,18 +49,13 @@ public void configure() {
   /**
    * Sample workflow. Schedules a dummy MR job.
    */
-  public static class SampleWorkflow implements Workflow {
+  public static class SampleWorkflow extends AbstractWorkflow {
 
     @Override
-    public WorkflowSpecification configure() {
-      return WorkflowSpecification.Builder.with()
-        .setName("SampleWorkflow")
-        .setDescription("SampleWorkflow description")
-        .startWith(new DummyAction())
-        .last(new DummyAction())
-        .addSchedule(new Schedule("Schedule", "Run every 2 seconds", "0/1 * * * * ?",
-                                  Schedule.Action.START))
-        .build();
+    public void configure() {
+        setName("SampleWorkflow");
+        setDescription("SampleWorkflow description");
+        addAction(new DummyAction());
```

Comment:
Yes. As a part of PR https://github.com/caskdata/cdap/pull/914/ we removed the redundant action from AppWithSchedule.java in the test directory, however missed this one which should also have been removed.